apps/web-giddh/src/assets/locale/user-details/en.json

```diff
@@ -48,7 +48,8 @@
         "signin_time": "Signin Time",
         "duration": "Duration (DD/HH/MM/SS)",
         "browser_agent": "Browser Agent",
-        "current": "Current"
+        "current": "Current",
+        "delete_all_session": "Are you sure you want to delete all the sessions,all the"
```

Comment:
Check and confirm message from Kriti and translate as well.

Reply:
DONE